tests/test_main.cpp

```diff
@@ -187,7 +187,7 @@ int test_main(int argc, char *argv[]) {
 	}
 	// Doctest runner.
 	doctest::Context test_context;
-	List<String> test_args;
+	LocalVector<String> test_args;
```

Comment:
Didn't touch almost any case of `List` but decided to do so here to avoid messy code, this PR doesn't generally look at broader decisions on the applicability of `List` over other structures, as that requires a lot more detailed evaluation of the code

Reply:
Yes there probably are a few cases where `LocalVector` would be better, but agreed better for separate PR.